docs/installation.md

```diff
@@ -94,8 +94,8 @@ sudo systemctl restart docker
 sudo systemctl show --property Environment docker
 ```
 
-## Docker-compose Installation
-To install a specific version of docker-compose (for example **_1.21.2_**) follow these steps:
```

Comment:
Same comment here - docker-compose is the name of the tool, not a stand-alone mention of docker.